app/routes/articles/show.tsx

```
@@ -0,0 +1,68 @@
+                <RemixHead>
+                  <title>{`${data.title} - DevRelKaigi 2025`}</title>
+                </RemixHead>
+                <Breadcrumb items={[
+                  { label: t('Home'), href: `/${locale}` },
+                  { label: t('Articles'), href: `/${locale}/articles` },
+                  { label: data.title! }
+                ]} />
+                <h1>{data.title}</h1>
+                <div
+                  className="article-body"
+                  dangerouslySetInnerHTML={{ __html: md.render(data.body as string) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)